pkgs/data/themes/ant-theme/ant-dracula.nix

```diff
@@ -1,34 +0,0 @@
-{ stdenv, fetchurl, gtk-engine-murrine }:
-
-let
-  themeName = "Ant-Dracula";
-in
-stdenv.mkDerivation rec {
-  pname = "ant-dracula-theme";
-  version = "1.3.0";
-
-  src = fetchurl {
-    url = "https://github.com/EliverLara/${themeName}/releases/download/v${version}/${themeName}.tar";
-    sha256 = "00b8w69xapqy8kc7zqwlfz1xpld6hibbh35djvhcnd905gzzymkd";
-  };
-
-  propagatedUserEnvPkgs = [
-    gtk-engine-murrine
-  ];
```

Comment:
Why is `gtk-engine-murrine` removed? It seems that the gtk2 theme still depends on this engine.

Reply:
Sorry, I checked the code, it uses this in gnome-terminal, which I don't use. I readd it.